src/alt/index.js

```diff
@@ -77,6 +42,51 @@ class Alt {
     return makeAction(this, 'global', name, implementation, obj)
   }
 
+  createUnsavedStore(StoreModel, ...args) {
+    StoreModel.config = fn.assign({
+      invisible: true
+    }, StoreModel.config)
+    return this.createStore(StoreModel, null, ...args)
+  }
+
+  createStore(StoreModel, iden, ...args) {
+    // pull all prototype methods out
+    const proto = utils.getInternalMethods(StoreModel, true)
+
+    // force inheritance on store
+    utils.inherit(StoreModel, Store, Object.keys(proto).reduce((obj, name) => {
```

Comment:
FWIW my thought with using the base `Store` effectively as a mixin is that you fully control that, so you know that it doesn't have any superclasses (or if it does, you know how they're set up), whereas you can't assume anything about whatever crazy thing the user passes in to `createStore`.